tests/test_config.py

```diff
@@ -183,13 +183,13 @@ def test_range_and_list_values(self):
                             'batch_sizes': [1],
                             'concurrency': [1]
                         },
-                        objectives=['perf_throughput']),
+                        objectives={'perf_throughput': 10}),
```

Comment:
What good does this do with a single metric? Are you just testing to see that the one with the highest through put is selected always?

Reply:
This particular unit test does not test anything to do with objectives, so it is just ensuring the default value is present.

Reply:
There are unit tests test_result_comparator.py that test with multiple metrics.